core/commands/add.go

```diff
@@ -242,7 +247,7 @@ You can now refer to the added file in a gateway, like so:
 			return
 		}
 
-		progress, _, err := req.Option(progressOptionName).Bool()
+		progress, prgFound, err := req.Option(progressOptionName).Bool()
 		if err != nil {
```

Comment:
Note: Okay so how is it that prgFound can be false, it we explicitly set it the PreRun.  I am very unsure of the correct behavior here.  What was the original intention.  @whyrusleeping @Kubuxu any ideas?